src/vorta/borg/borg_job.py

```diff
@@ -277,6 +277,9 @@ def read_async(fd):
 
                         elif parsed['type'] == 'file_status':
                             self.app.backup_log_event.emit(f'{parsed["path"]} ({parsed["status"]})', {})
+                        elif parsed['type'] == 'progress_percent':
+                            if parsed.get("message"):
```

Comment:
I skipped json lines with no message because I didn't know what to do with them. They looked like this:
`{'operation': 1, 'msgid': 'repository.check', 'type': 'progress_percent', 'finished': True, 'time': 1671484754.281054}`